processor/routingprocessor/router.go

```diff
@@ -418,7 +439,7 @@ func (r *router) registerExportersForDefaultRoute(available ExporterMap) error {
 		v, ok := available[exp]
 		if !ok {
 			return fmt.Errorf("error registering default exporter %q: %w",
-				exp, errExporterNotFound,
+				exp, errDefaultExporterNotFound,
```

Comment:
I'm wondering if instead of returning the error, warning could be logged in this function and the whole block in `registerExports` could be kept much simpler?

Reply:
Sounds reasonable. Let me try that.

Reply:
The advantage of doing it as part of `registerExporters` is that we can tell the user what's the pipeline type that is being affected. I think this better user experience is worth the extra complexity, which isn't that complex anyway. What do you think?

Reply:
I agree. As another option, the signal could be provided as argument for this function. It adds some extra code as well though (maybe it's a bit simpler, but the change is in more places, so I am not convinced if it's really better)

Reply:
I thought about it as well, but it doesn't feel right to pass it only for logging purposes.